### DISCUSSION TITLE
Configuration of "push" sensors (export/voltage/production)

### DISCUSSION THREAD

**Discussion by HLFCode** [firstof9/openevse#318] [Q&A] [answered]
Thanks for the excellent integration.
How do you kick off the push sensors?
I've configured them in the HA integration config (where is says "You can also enter your grid import/export or solar production sensors to push data to OpenEVSE.) but nothing seems to be getting to OpenEVSE.
I've also monitored MQTT and nothing is being posted to the topics defined in the OpenEVSE config.
I've proved OpenEVSE/MQTT works by temporarily testing it with an HA automation to post the sensor values.
Thanks

**Comment by firstof9** [comment, marked as answer]
> How do you kick off the push sensors?

When you configure a sensor, it gets monitored and pushed to the OpenEVSE via the HTTP API. I tested this myself and do see the sensors updating as expected in OpenEVSE.

**Comment by HLFCode** [comment]
Weird
I have a sensor `sensor.myenergi_hlf_power_generation` which I pasted into the `Solar Production (optional)` field.
The OpenEVSE is set for `Auto Mode`, `Eco `
Self Production is `Enabled ` and mode is `Export ` and I've deleted the MQTT topic.
Still not updating in the charger as the `Production `value remains steadfastly at 0A.
I'm using the latest WfFi firmware (V5.1.0) could that be a problem?

**Reply by firstof9** [reply]
> I'm using the latest WfFi firmware (V5.1.0) could that be a problem?

Shouldn't be that's what I'm using. The value is pushed to `grid_ie` in OpenEVSE.

**Comment by HLFCode** [comment]
I must have done something stupid because it's working now.
Thanks for the integration and your help
